src/components/features/forms/NewMarkerForm.tsx

```diff
@@ -1,43 +1,36 @@
 import { Controller, UseFormReturn } from 'react-hook-form'
-import { Checkbox, FormControlLabel, TextField } from '@mui/material'
+import { TextField } from '@mui/material'
 import { useTranslation } from 'react-i18next'
 import { NewMarkerFormData } from '@/schemas/newMarkerSchema'
 
 interface NewMarkerFormProps extends UseFormReturn<NewMarkerFormData> {
-  isFinal: boolean
-  setIsFinal: React.Dispatch<React.SetStateAction<boolean>>
+  defaultValue?: string
```

Comment:
lets write `defaultValue?: MarkerFormDto`

and in default values you will do 

`defaultValue?.name || ''`

because if we have more properties we dont want to pass them in different props but rather just send the whole object.

Reply:
![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.fe/assets/123353314/220588cf-d913-4ce0-a228-13e8856b17ae)
![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.fe/assets/123353314/cf0dc44b-b6be-47ae-a4ef-bc24b262d5cc)